.github/workflows/dockerimage-ci.yml

```diff
@@ -24,8 +24,8 @@ jobs:
     name: Code checks
```

Comment:
I think we can simplify the Github workflow a bit:

I find the value of the cron job questionable. I suggest disabling that.
We can merge build_pr and build_edge into a single job. Furthermore we don't need to push to the container registry. I think it's fine if we push on release only. These jobs can instead publish the built images as artifacts.